package.json

```diff
@@ -11,6 +11,8 @@
     "config",
     "index.d.ts"
   ],
+  "main": "app.js",
```

Comment:
egg plugin don't need `main` usually because it is loaded by egg-loader instead of `require`.